ivy/data_classes/array/layers.py

```diff
@@ -1,6 +1,6 @@
 # global
 import abc
-from typing import Optional, Tuple, Union, List, Sequence, Dict
+from typing import Optional, Tuple, Union, List, Sequence
```

Comment:
I don't see what this has to do with your changes, was this from the pre-commit?